docs/tutorials/fsi-montecarlo-on-batch/README.md

```diff
@@ -0,0 +1,201 @@
+* The output of the Monte Carlo simulation is published to a [PubSub](https://cloud.google.com/pubsub/docs/overview) topic.
+* The PubSub data is entered into [BigQuery](https://cloud.google.com/bigquery) 
+via a [PubSub BigQuery subscription](https://cloud.google.com/pubsub/docs/bigquery)
+* The data is visualized via a [Vertex AI Workbench](https://cloud.google.com/vertex-ai-workbench) 
+[Jupyter Notebook](https://jupyter.org/) 
+
+<img src="./images/architecture.png" width="800" />
+
+## Basic getting started
+
+1. Get the HPC Toolkit configured.
+
+Build the `ghpc` binary:
+
+```shell
+git clone https://github.com/GoogleCloudPlatform/hpc-toolkit
+cd hpc-toolkit
+make
+./ghpc --version
+./ghpc --help
+```
+
+2\. Enable Google Cloud Resource Manager:
+
+```
+gcloud services enable cloudresourcemanager.googleapis.com
+```
+
+3\. Run `ghpc` on the blueprint `fsi-montecarlo-on-batch.yaml`
+
+```
+./ghpc create community/examples/fsi-montecarlo-on-batch.yaml
+```
+
+If successful, you will see output similar to:
+```
+To deploy your infrastructure please run:
+
+./ghpc deploy fsimontecarlo
+```
+4\. Edit the file to update `<my_project>` with your project_id:
+
+```
+  project_id: <my_project>
+```
+
+5\. Deploy the blueprint as instructed:
+
+```
+./ghpc deploy fsimontecarlo
+```
+If successful, this will prompt you:
+```
+Summary of proposed changes: Plan: 22 to add, 0 to change, 0 to destroy.
+(D)isplay full proposed changes,
+(A)pply proposed changes,
+(S)top and exit,
+(C)ontinue without applying
+Please select an option [d,a,s,c]:
+```
+Respond with `apply`, "a". You may be asked to respond twice. 
+
+When the job is complete it will indicate `Succeeded`. You can then proceed to the next section.
+
+At this point, all the required infrastructure has been deployed.
+
+## Open Vertex AI Workbench
+
+1. Go to the Vertex AI Workbench Notebooks instances in the Google Cloud Console:
+```
+https://console.cloud.google.com/vertex-ai/workbench/user-managed
+```
+2. Open JupyterLab on the Notebook instance listed.
+
+<img src="./images/workbench.png" width="500" />
+
+```
+Click on `OPEN JUPYTERLAB` link
+```
+
+3. In the JupyterLab UI, you will see a list of directories:
+```
+Select `fsi`
+```
+Under `fsi` all the files required to run the demo have been pepared.
+
+<img src="./images/files.png" width="300" />
+
+4. Open a terminal window by clicking on the terminal icon.
+
+<img src="./images/terminal.png" width="200" />
+
+4. Update the local Python requirements:
+```
+python3 -m pip install --user -r requirements.txt
+```
+There may be some incompatibilities listed, but it will not affect this demo.
+
+5. Run the `batch.py` Python script to ensure it is working.
+```
+python3 batch.py --help
+``` 
+You will see a listing of the help messages
+
+6. To start the VaR simulation, run `batch.py` with the config file and `--create_job`
+```
+python3 batch.py --config_file mc_run.yaml --create_job
```

Comment:
We should probably mention cost somewhere before we have the user kick off a job.

Reply:
Done